src-test/org/etools/j1939_84/bus/j1939/packets/SupportedSPNTest.java

```diff
@@ -15,6 +17,15 @@
  */
 public class SupportedSPNTest {
 
+    @Test
+    public void testEquals() {
+        SupportedSPN instance = new SupportedSPN(new int[] { 0x01, 0x02, 0x1D, 8 });
```

Comment:
This test should verify that two different instances of SupportedSPN are not the equal because the members that make up the equal functions are different.  Ideally, this would test each member individually, but that might be difficult here.